samplekotlin/src/main/java/com/example/chattutorial/ChannelActivity.kt

```diff
@@ -44,33 +43,32 @@ class ChannelActivity : AppCompatActivity(R.layout.activity_channel) {
                 is MessageListViewModel.Mode.Normal -> messageInputViewModel.resetThread()
             }
         }
+
+        messageListViewModel.state.observe(this) {
```

Comment:
Had to add this to get back to work. I think it's taken care of in another PR. I wonder if we can consolidate `Mode` into `State`?